icebox/icebox.py

```diff
@@ -2124,6 +2269,17 @@ def parse_db(text, device="1k"):
         (12,  0, 1), #6 fixed
         (12, 31, 1), #7 fixed
     ],
+    "u4k": [
+        (19,  0, 1), # 0 ok
+        ( 6,  0, 1), # 1 ok
+
+        (13, 21, 0), # 2 ok
+        (19, 21, 0), # 3 unclear
+        (12, 21, 0), # 4 HFOSC unclear
+        (12,  0, 0), # 5 LFOSC unclear
+        ( 6,  0, 0), # 6 unclear
```

Comment:
There aren't normally two padins in the same tile. Based on the up5k, this is probably (6, 21, 0) - probably best to fill the other unclear ones in from the up5k too substituting 31 with 21.

Reply:
fixed